x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java

```diff
@@ -265,4 +284,45 @@ public String getFeatureName() {
     public String getFeatureDescription() {
         return "Manages data related to Enrich policies";
     }
+
+    /**
+     * A class that specifies either a flat (unit-less) number or a byte size value.
+     */
+    public static class FlatNumberOrByteSizeValue {
```

Comment:
Any suggestions for this name are welcome :sweat_smile: (also it's location if people think it should be a separate class and/or in a different package).

Reply:
I think it's okay here. I can't think of a better name either